white_threshold: 200
black_threshold: 10
monochrome_threshold: 20